cli/src/commit_templater.rs

```diff
@@ -2528,6 +2577,93 @@ fn builtin_diff_stats_methods<'repo>() -> CommitTemplateBuildMethodFnMap<'repo,
     map
 }
 
+fn builtin_diff_stats_entry_methods<'repo>() -> CommitTemplateBuildMethodFnMap<'repo, DiffStatEntry>
+{
+    // Not using maplit::hashmap!{} or custom declarative macro here because
+    // code completion inside macro is quite restricted.
+    let mut map = CommitTemplateBuildMethodFnMap::<DiffStatEntry>::new();
+    map.insert(
+        "path",
+        |_language, _diagnostics, _build_ctx, self_property, function| {
+            function.expect_no_arguments()?;
+            let out_property = self_property.map(|entry| entry.path.target);
+            Ok(out_property.into_dyn_wrapped())
+        },
+    );
+    map.insert(
+        "status",
+        |_language, _diagnostics, _build_ctx, self_property, function| {
+            function.expect_no_arguments()?;
+            let out_property = self_property.map(|entry| {
+                let status = match entry.status {
+                    DiffEntryStatus::Copied => "C",
+                    DiffEntryStatus::Renamed => "R",
+                    DiffEntryStatus::Modified => "M",
+                    DiffEntryStatus::Added => "A",
+                    DiffEntryStatus::Removed => "D",
+                };
+                status.to_owned()
+            });
+            Ok(out_property.into_dyn_wrapped())
```

Comment:
Better to use long form as we do for `TreeDiffEntry`.

We can add `status_char()` if needed.